Handlers/Chat.py

```diff
@@ -17,6 +17,12 @@ def get_chat(self, cid):
     def get_chat_messages(self, cid):
         return self.chatDAO.get_chat_messages(cid)
 
+    def get_chat_members(self, cid):
+        return self.chatDAO.get_members_from_chat(cid)
+
+    def get_chat_owner(self, cid):
```

Comment:
There's no need to get chat owner.

By visiting /chat/<int:cid> You will get all info pertaining to chat, including chat owner